pages/events/index.vue

```diff
@@ -75,6 +75,9 @@
             :to="event.to"
           />
         </div>
+        <div v-else class="event-page__results">
+          <h3>Plan an event in your area today!</h3>
```

Comment:
This copy is just information and I don't think it has a title connotation. Could you change to a paragraph, please?